cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -93,6 +93,7 @@ static int _request_transmit(ieee802154_dev_t *dev)
 
     if (cc2538_csma_ca_retries < 0) {
         RFCORE_SFR_RFST = ISTXON;
+        RFCORE_XREG_CSPCTRL |= CC2538_CSP_MCU_CTRL_MASK;
```

Comment:
can you add a short comment as to what that does?

Reply:
sure!

Reply:
done!

Reply:
A `/* disable CSP Strobe Processor */`would have been fine too, but this is even better.